docs/modules/ROOT/pages/interfaces.adoc

```diff
@@ -0,0 +1,177 @@
+= Interfaces and Dispatchers
+
+This section describes the interfaces OpenZeppelin Contracts for Cairo offer, and explains the design choices behind them.
+
+Interfaces can be found in the module tree under the `interface` submodule, such as `token::erc20::interface`. For example:
+
+```javascript
+use openzeppelin::token::erc20::interface::IERC20Dispatcher;
+```
+
+or
+
+```javascript
+use openzeppelin::token::erc20::dual20::DualERC20Trait;
+```
+
+NOTE: For simplicity, we'll use ERC20 as example but the same concepts apply to other modules.
+
+== Interface types
```

Comment:
```suggestion
== Traits for representing Interfaces
```

Reply:
we use traits in many ways throughout the codebase. this section is focused on interfaces: ABIs, standard interfaces, and dispatchers are traits meant to _interface_ with contracts whose interface is unknown. it's all interface-oriented. we don't care about other kind of traits, or traits themselves since they're just the language mechanism we're using to deal with interfaces. in other contexts, say a rust CLI or javascript website, interfaces can be deal with other stuff than traits.

Reply:
what's the problem with `Interface types`?

Reply:
went with `Interface traits` for the section and `Standard traits` for `IERC20`

Reply:
> Trait types is not interface-oriented. Remember these traits aim to interface with smart contracts -- we don't care about other kind of traits.

and

> what's the problem with Interface types? maybe Interface traits works better?

The issue for me consisted of the three listed `interface type` titles being called `traits`. `Interface traits` I think follows better

----

>> The next section after Interface Types is Dual Interfaces. It's not the same interface type that we're talking about in the former, but it is another type of interface (if that makes sense)

> Dual interfaces are just about the design of specific interfaces/what includes, not actually a type (ABI/standard/dispatcher).

Right, I totally get that. I was just pointing out a potential area of confusion. If we're using `Interface traits`, then my point is moot

Reply:
>> The next section after Interface Types is Dual Interfaces. It's not the same interface type that we're talking about in the former, but it is another type of interface (if that makes sense)

> Dual interfaces are just about the design of specific interfaces/what includes, not actually a type (ABI/standard/dispatcher).

Right, I totally get that. I was just pointing out a potential area of confusion. If we're using `Interface traits`, then my point is moot